lib/mix/tasks/phoenix.gen.model.ex

```diff
@@ -87,11 +93,12 @@ defmodule Mix.Tasks.Phoenix.Gen.Model do
 
     Mix.Phoenix.check_module_name_availability!(binding[:module])
 
+    {uniques, attrs} = extract_uniques(attrs)
```

Comment:
What if we change `attrs` to return `{attrs, uniques}`? I think it will cause less changes to the code, otherwise we need to filter out the `:unique` in a bunch of different places like we do today. :)